db/migrate/20210226185223_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# This migration comes from active_storage (originally 20170806125915)
+class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
```

Comment:
Why do we need ActiveStorage tables if we're using S3? I'd have assumed one or the other (and I'd probably go with the DB for simplicity for now, personally)